On version numbering, from discussion in PR #34535

marbx opened the pull request:
### What does this PR do?
Documenting the relationship between version numbering and Git 

From discussion in PR #34535

marbx committed:
Optional repo-cache for offline/faster compile

marbx committed:
Documentation and and catching lost PR 38794

marbx committed:
Documentation and formatting

marbx committed:
On version numbering, from discussion in PR #34535

marbx committed:
Merge branch 'develop' into Optional_repo_cache_for_offline_faster_compile

marbx committed:
Merge pull request #1 from markuskramerIgitt/Optional_repo_cache_for_offline_faster_compile

Yes

marbx committed:
Merge branch 'develop' of https://github.com/saltstack/salt into develop

marbx committed:
Replace bitsadmin transfer with DownloadFileWithProgress

marbx committed:
replace bitsadmin transfer with DownloadFileWithProgress

cachedout commented on doc/topics/releases/version_numbers.rst:
```diff
@@ -46,3 +46,50 @@ to be cut and the month is ``June``. This would make the new release number
 code name and the process repeats itself.
+
+
+Version numberss, Git and salt --version
```
`Version numbers`

cachedout commented on doc/topics/releases/version_numbers.rst:
```diff
@@ -46,3 +46,50 @@ to be cut and the month is ``June``. This would make the new release number
+
+Version numberss, Git and salt --version
+-------
+The salt version, for prorgammers, is based on `git describe` and presented to end users with `salt --version`.
```
`programmers`

cachedout replied:
`end-users`

marbx committed:
Corrections of @cachedout

markuskramerIgitt commented:
Hello @cachedout, thank you for your corrections.

I fxied my spelling errors and checked in 33c1978, which appears above.
Does this mean I don't need another Pull Request?

marbx committed:
Reformat (replace whitespace with tab)

marbx committed:
Reformat (replace whitespace with tab)

marbx committed:
Reformat (replace whitespace with tab)

marbx committed:
Merge pull request #2 from markuskramerIgitt/replace_bitsadmin

Replace bitsadmin -- to me

marbx committed:
Merge pull request #3 from markuskramerIgitt/Optional_repo_cache_for_offline_faster_compile

Reformat (replace whitespace with tab) -- to me

marbx committed:
Merge branch 'develop' of https://github.com/saltstack/salt into develop

marbx committed:
Replaced bitsadmin with DownloadFileWithProgress
because bitsadmin not availlable and
DownloadFileWithProgress allow offline compile

marbx committed:
Merge branch 'develop' into replace_bitsadmin

marbx committed:
Merge pull request #4 from markuskramerIgitt/replace_bitsadmin

Replaced bitsadmin with DownloadFileWithProgress

rallytime commented on doc/topics/releases/version_numbers.rst:
```diff
@@ -46,3 +46,50 @@ to be cut and the month is ``June``. This would make the new release number
+  +------------+----------------------------------------------------------------------------+
+  | 2016.11    | **Release branch** Actively developed bug-fixes for all 2016.11.* releases |
+  +------------+----------------------------------------------------------------------------+
+  | v2016.11.1 | Tag of the moment that the 2016.11.1 branch was created                    |
```
Change the word `branch` to `release` here. The `v2016.11.1` syntax is not a branch. It is a tag signaling the commit that a release is based on.

markuskramerIgitt replied:
@rallytime, thank you, I changed the wording (on my local file).

In fact there are many unrelated files in this pull request. This was not my intention.
I will now make a local backup of my pending changes, cancel all my pull requests, delete ally my branches and start over from scratch.

rallytime reviewed (changes requested):
I had one small technicality about the version_numbers doc changes. Once that is fixed, I think the doc changes look good. Thanks for putting this together @markuskramerIgitt!

However, I did not review the other files included here, as the windows build files are not my area. Were those files meant to be included here?
